src/components/QueryEditor/Stack.tsx

```diff
@@ -11,7 +11,9 @@ interface StackProps {
   wrap?: boolean;
   gap?: number;
 }
-
+/**
+ * @deprecated use the EditorStack component. Available starting from grafana-ui@10.2.3
+ */
 export const Stack = ({ children, ...props }: StackProps) => {
```

Comment:
Is it possible to just replace the component with the new implementation or is it too risky?

Reply:
I am curious as well. Although I also kind of like the fact, that we are distinguishing `EditorStack` from `Stack` from `grafana-ui`.

Reply:
We could, but then we should probably increment plugin-ui's major version, as this would be a breaking change for plugins using this component